src/sentry/notifications/notifications/rules.py

```diff
@@ -151,6 +152,19 @@ def send(self) -> None:
         shared_context = self.get_context()
 
         for provider, participants in participants_by_provider.items():
+            if self.target_type == ActionTargetType.RELEASE_MEMBERS:
+                last_release = shared_context["last_release"]
+                release_version = last_release.version if last_release else None
+                for participant in participants:
+                    active_release_notification_sent.send_robust(
```

Comment:
@snigdhas if all you’re going to do in your receiver is call an analytics function, you can just call that directly here. It’s a synchronous call no matter what.